akismet-privacy-policies.php

```diff
@@ -86,6 +110,30 @@ public static function get_object() {
 		return self::$classobj;
 	}
 
+	/**
+	 * Initialize and translate $this->notice, $this->error_message
+	 *
+	 * @since x.x.x
+	 * @access public
+	 *
+	 * @return void
+	 */
+	public function translate_strings() {
+		// dummy translation, only there to let poedit recognize the text...
+		__( 'I accept that my given data and my IP address is sent to a server in the USA only for the purpose of spam prevention through the <a href="http://akismet.com/">Akismet</a> program.<br /><a href="https://akismet.com/gdpr/">More information on Akismet and GDPR</a>.', 'akismet-privacy-policies' );
```

Comment:
We need escaping so we haven' control about the content inside the language files, so please switch for the example from `__()` to `esc_html__()`.

Reply:
Done